src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -4797,14 +4712,17 @@ SafeContext getIndexerEscape(
 
                 var escapeValues = ArrayBuilder<EscapeValue>.GetInstance();
                 GetEscapeValues(
-                    methodInfo,
-                    // This is calculating the actual receiver scope
-                    null,
-                    ThreeState.Unknown,
-                    methodInfo.Method.Parameters,
-                    expr.Arguments,
-                    expr.ArgumentRefKindsOpt,
-                    expr.ArgsToParamsOpt,
+                    new MethodInvocationInfo
+                    {
+                        MethodInfo = methodInfo,
+                        ReceiverOpt = null,
```

Comment:
Consider preserving the comment "// This is calculating the actual receiver scope"